tests/sys/vane/ford.hoon

```diff
@@ -6999,6 +6999,195 @@
     results4
     (expect-ford-empty ford-gate ~nul)
   ==
+::
+++  test-vega-empty  ^-  tang
+  ::
+  =^  results  ford-gate
+    %-  ford-call  :*
+      ford-gate
+      now=~1234.5.8
+      scry=scry-is-forbidden
+      ::
+      call-args=[duct=~[/wherever] type=~ %vega ~]
+      ::
+      moves=~
+    ==
+  (weld results (expect-ford-empty ford-gate ~nul))
```

Comment:
`+expect-ford-empty` isn't quite good enough as a test case here, because it clears Ford's caches: https://github.com/urbit/arvo/pull/1070/files#diff-bae0fc44604de49eb6a2fb9a968d81a9L7352

Probably the thing to do is make a new similar arm that doesn't send the `%keep` move.